content/courses/program-security/signer-auth.md

```diff
@@ -299,6 +299,11 @@ While this is somewhat contrived in that any DeFi program with a vault would be
 more sophisticated than this, it will show how the lack of a signer check can
 result in tokens being withdrawn by the wrong party.
 
+additional checks ensure:
+a. The withdraw amount is not zero.
+b. There are sufficient funds in the account.
+c. The withdraw destination is not the same as the vault's token account.
+
```

Comment:
We're trying to focus on just signer checks in this chapter. I'm not totally sure adding other checks helps.